app/build/plugins/image/index.js

```diff
@@ -11,43 +13,58 @@ function render ($, callback, options) {
   // Process 5 images concurrently
   eachEl($, 'img', function(el, next){
 
-    var src = $(el).attr('src');
+    // Decode any doubly-encoded ampersands in the image src
+    var src = decodeAmpersands($(el).attr('src'));
     var width, height;
 
-    debug(src, 'checking cache');
+    // Test for query string to skip caching & optimizing
+    var parsedSrc = url.parse(src, { parseQueryString: true });
+    debug(src, 'is parsed into', parsedSrc)
 
-    cache.lookup(src, optimize(blogID), function(err, info){
+    if (parsedSrc.query.static) {
+      debug(src, 'Image has \'static\' URL param, skipping');
+      return next();
+    }
+
+    // No URL params or hash values
+    var cleanSrc = origin(parsedSrc) + parsedSrc.pathname;
+    debug(src, 'is cleaned into', cleanSrc);
+
+    debug(cleanSrc, 'checking cache');
+
+    // Pass in the `pathname` component of the image src (no URL params or hash)
+    cache.lookup(cleanSrc, optimize(blogID), function(err, info){
```

Comment:
What are your thoughts on cleaning the src inside the Transformer?  Again, doing so will apply this fix to the thumbnail generator as well as the image cacher

Reply:
That would make sense.

Reply:
This should work for now